tests/e2e/testdata/scenarios/fields/gkehubfeaturemembership/set_unset/script.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: containercluster-${uniqueId}
+spec:
+  location: us-central1-a
+  workloadIdentityConfig:
+    workloadPool:  mock-project.svc.id.goog
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeature
+metadata:
+  name: gkehubfeature-${uniqueId}
+spec:
+  projectRef:
+    external: projects/mock-project
+  location: global
+  resourceID: configmanagement
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubMembership
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: mock-project
+  name: gkehubmembership-${uniqueId}
+spec:
+  location: global
+  authority:
+    issuer: https://container.googleapis.com/v1/projects/mock-project/locations/us-central1-a/clusters/containercluster-${uniqueId}
+  description: A sample GKE Hub membership
+  endpoint:
+    gkeCluster:
+      resourceRef:
+        name: containercluster-${uniqueId}
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  name: gkehubfeaturemembership-${uniqueId}
+  annotations:
+    cnrm.cloud.google.com/state-into-spec: absent
+spec:
+  projectRef:
+    external: projects/mock-project
+  location: global
+  membershipRef:
+    name: gkehubmembership-${uniqueId}
+  featureRef:
+    name: gkehubfeature-${uniqueId}
+  configmanagement:
+    configSync:
+        sourceFormat: unstructured
```

Comment:
Could you update the indentation of this line?

Reply:
Done